daffodil-test/src/test/resources/org/apache/daffodil/section12/delimiter_properties/DelimiterProperties.tdml

```diff
@@ -111,6 +111,15 @@
       </xs:complexType>
     </xs:element>
 
+    <xs:element name="DP_04_02" dfdl:lengthKind="implicit">
+      <xs:complexType>
+        <xs:sequence>
+          <xs:element name="s1" type="xs:float"
+            dfdl:terminator=" %WSP+;" maxOccurs="unbounded" />
+        </xs:sequence>
```

Comment:
Might be good to have a trailing space check as well. That should also be an SDE, since the user probably meant ``"FOO%SP;"`` rather than ``"FOO "``